packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -5,7 +5,11 @@ module.exports = {
     Map: "map",
     WeakMap: "weak-map",
     Set: "set",
-    WeakSet: "weak-set"
+    WeakSet: "weak-set",
+    Dict: "dict",
+    setImmediate: "set-immediate",
+    clearImmediate: "clear-immediate",
+    delay: "delay"
```

Comment:
Yep, only `Dict` and `delay`. All other non-standard / no-proposal `core-js` features already in `runtime`. I do not think it makes sense do an exception only for them.